### PR TITLE
Fix generating operation string for nested inline fragments

### LINKED ISSUE #202: GQL operation string is missing fragment used in a query
For the given GQL schema:
```graphql
type ItemA {
  valueA: String
}

type ItemB {
  valueB: String
}

union Item = ItemA | ItemB

type Container {
  name: String
  items: [Item]
}

type Query {
  container(id: ID!): Container
}
```

and following query:
```graphql
fragment ItemAFragment on ItemA {
  valueA
}

fragment ItemBFragment on ItemB {
  valueB
}

fragment ItemFragment on Item {
  ... on ItemA {
    ...ItemAFragment
  }
  ... on ItemB {
    ...ItemBFragment
  }
}

fragment ContainerFragment on Container {
  name
  items {
    ...ItemFragment
  }
}

query getContainer($id: ID!) {
  container(id: $id) {
    ...ContainerFragment  
  }
}
```

the generated client contains the following `get_container` method:
```python
...
    async def get_container(self, id: str) -> GetContainer:
        query = gql(
            """
            query getContainer($id: ID!) {
              container(id: $id) {
                ...ContainerFragment
              }
            }

            fragment ContainerFragment on Container {
              name
              items {
                ...ItemFragment
              }
            }

            fragment ItemFragment on Item {
              ... on ItemA {
                ...ItemAFragment
              }
              ... on ItemB {
                ...ItemBFragment
              }
            }
            """
        )
...
```

Operation string is missing `ItemAFragment` and `ItemBFragment` definitions, even though they are used in `ItemFragment`.

### PR DESCRIPTION
**Pull request by jakub-bacic**
Resolves #202

### CODE REVIEW

File: CHANGELOG.md
Comment (rafalp) [inline CHANGELOG.md:6]:
```suggestion


```

**Review by rafalp — APPROVED** [review]

**Comment by jakub-bacic** [comment]
Removed changes applied to CHANGELOG as suggested :).

**Comment by mat-sop** [comment]
> Removed changes applied to CHANGELOG as suggested :).

I think, the suggestion was to leave 2 empty lines between the new entry and `## 0.8.0 (2023-08-22)`, not to remove the entry :)

**Comment by jakub-bacic** [comment]
> > Removed changes applied to CHANGELOG as suggested :).
> 
> I think, the suggestion was to leave 2 empty lines between the new entry and `## 0.8.0 (2023-08-22)`, not to remove the entry :)

Oh yeah, I think you're right. My bad 🤦 Fixed again but properly this time :)

BTW, who's the maintainer of the repo? GH says that 1 workflow is still awaiting approval.

**Comment by mat-sop** [comment]
I approved the workflow and merged this pr. Thank you for contributing :)

**Pull request merged**
